test/functional/apps/discover/_discover.js

```diff
@@ -156,9 +158,9 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.visualize.waitForVisualization();
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        if (newDurationHours < 1 || newDurationHours >= 5) {
-          throw new Error(`expected new duration of ${newDurationHours} hours to be between 1 and 5 hours`);
-        }
+        expect(Math.round(newDurationHours)).to.be(3);
+        const rowData = await PageObjects.discover.getDocTableIndex(1);
+        expect(rowData).to.have.string('Sep 20, 2015 @ 02:51:53.327');
```

Comment:
:information_source: There is some slight difference to master here in the document that is retrieved. I checked and brushing still works as expected just the document is (deterministic) a different one from master and 7.x (I assume some other changes outside this code). So I adjusted this test for the 7.3 backport to match the document ont he 7.3 branch.